Makefile

```
@@ -780,6 +780,10 @@ aks-create: $(KUBECTL) ## Create aks cluster as mgmt cluster.
 	./scripts/aks-as-mgmt.sh
 	MANIFEST_IMG=$(CONTROLLER_IMG) MANIFEST_TAG=$(TAG) $(MAKE) set-manifest-image
 
+.PHONY: aks-reset
+aks-reset: $(KUBECTL) ## Deletes the resource group and the associated AKS cluster created by aks-create.
```

I wonder if it is worth renaming the target to `aks-delete` instead of `aks-reset` since we are literally trying to delete the AKS clusters. 
```suggestion
.PHONY: aks-delete
aks-delete: $(KUBECTL) ## Deletes the resource group and the associated AKS cluster created by aks-create.
```